src/luxonis_ml/loader/loader.py

```diff
@@ -100,7 +100,7 @@ def load_image_with_annotations(self, idx: int):
         if self.stream and self.dataset.bucket_storage.value != "local":
             img_path = str(Path.home() / ".luxonis_mount" / path[1:])
         else:
-            img_path = f"{str(Path.home())}/.cache/luxonis_ml/data/{path}"
+            img_path = str(Path.home() / ".cache" / "luxonis_ml" / "data" / path[1:])
```

Comment:
Wouldn't it be safer to do the path construction with os.path.join?

Reply:
I think this is also safe. It would be nice to stick to one or the other though maybe but not sure.